lib/timer.py

```diff
@@ -20,12 +21,22 @@
 # OrderedDict is used to ensure each job is run sequentially in the order
 # listed as some jobs may depend on completion of the previous job.
 UACLIENT_JOBS = collections.OrderedDict(
-    {
-        "update_messaging": {
-            "job_func": update_apt_and_motd_messages,
-            "run_interval_seconds": 43200,  # 12 hours
-        }
-    }
+    [
+        (
+            "update_messaging",
+            {
+                "job_func": update_apt_and_motd_messages,
+                "run_interval_seconds": 43200,  # 12 hours
```

Comment:
can we cut these jobs in half and do 6hrs for apt/motd and 12 for status please?

Reply:
The initial idea was for the status job to run every 12h, but @blackboxsw suggested that we increased it to 24h, since the there is not many gap situation for the root vs non-root job.

I think we have the same logic for the `update_messaging` job where the job will only potentially produce new messages if an apt update operation occurred and changed the system state.

But if you think we should make the change, np

Reply:
Yea, I'd like to take advatange of these jobs to keep things fresher. It should be pretty light and we're not running that much.